### DISCUSSION TITLE
What about the service updates?

### DISCUSSION THREAD

**Discussion by sreydray** [seerge/g-helper#2633] [Q&A] [answered]
I want to know how the armoury crate provided updates(like refreshrateservice, corecomponent, etc) would be received if I delete armoury crate and use G-helper.

**Comment by seerge** [comment, marked as answer]
@SreyasXd AC "updates" are needed only for AC to work :) G-Helper doesn't need any of them

**Reply by sreydray** [reply]
ok, I used it. Nice app, does all the job of armoury crate and myasus. Thanks
